tasks/web-nginx.yml

```diff
@@ -42,4 +42,7 @@
     name: nginx
     state: started
 
-# SELinux should be handled by openmicroscopy.omero-web-runtime
```

Comment:
I think we may not need this change as it is already there
[ name: omero web | selinux ports](https://github.com/ome/ansible-role-omero-web/blob/500e9ef5a2e8315793cd51badd97d0a22cf53958/tasks/web-dependencies.yml#L15C21-L15C28)
But I think the issue comes from the `selinux_utils` role as I think the `selinux_enabled `variable value does not reflect the `SeLinux `status so this task has been never run.

I think it is better to check the `SELinux `status on  [this task](https://github.com/ome/ansible-role-omero-web/blob/500e9ef5a2e8315793cd51badd97d0a22cf53958/tasks/web-dependencies.yml#L22) this task using the `ansible facts`, i.e. 
`when: ansible_facts.selinux.status == 'enabled'`
instead of 
`when: selinux_enabled`

Reply:
So should `httpd_can_network_connect` be added to the booleans set in https://github.com/ome/ansible-role-omero-web/blob/500e9ef5a2e8315793cd51badd97d0a22cf53958/tasks/web-dependencies.yml#L3-L12 instead?

This might also fix the idempotence failure

Reply:
I think what is needed is deleting [the added task](https://github.com/dominikl/ansible-role-omero-web/blob/58662ffdd0509e38dc74ad4836d54deb472f4dec/tasks/web-nginx.yml#L45), and checking the `SeLinux `status using ansible facts. I have made these changes in this [local branch ](https://github.com/khaledk2/ansible-role-omero-web/commit/e41e414664e40c3f01c52165a43ef0de4e92c304). It has passed the molecule tests, I think Dom should apply  these changes to his branch and test on a Rocky Linux deployment.

Reply:
👍 Sounds good, thanks @khaledk2 !